server/src/main/java/com/objectcomputing/checkins/services/feedback/Feedback.java

```diff
@@ -71,22 +71,23 @@ public Feedback(@NotNull String content,
                     @NotNull Boolean confidential,
                     @NotNull LocalDateTime createdOn,
                     @Nullable LocalDateTime updatedOn) {
-        this(null, content, sentTo, sentBy, confidential, createdOn, updatedOn);
+        this.id = null;
+        this.content = content;
+        this.sentTo = sentTo;
+        this.sentBy = sentBy;
+        this.confidential = confidential;
+        this.createdOn = createdOn;
+        this.updatedOn = updatedOn;
     }
 
     public Feedback(@Nullable UUID id,
                     @NotNull String content,
-                    @NotNull UUID sentTo,
-                    @NotNull UUID sentBy,
                     @NotNull Boolean confidential,
-                    @NotNull LocalDateTime createdOn,
-                    @Nullable LocalDateTime updatedOn) {
+                    @Nullable LocalDateTime updatedOn
+                    ) {
         this.id = id;
         this.content = content;
-        this.sentTo = sentTo;
-        this.sentBy = sentBy;
         this.confidential = confidential;
-        this.createdOn = createdOn;
         this.updatedOn = updatedOn;
```

Comment:
This can actually get removed too, can't it?

Reply:
This field was remove and date annotations added.